genie-core/src/main/java/com/netflix/genie/core/util/MetricsUtils.java

```diff
@@ -80,4 +83,44 @@ private MetricsUtils() {
     public static Map<String, String> newFailureTagsMapForException(final Throwable t) {
         return addFailureTagsWithException(Maps.newHashMap(), t);
     }
+
+    /**
+     * Convenience method to add common tags for slicing and dicing atlas metrics from Job
+     * execution environment to the map of tags passed in.
+     * Populates defaultIdValue in case ids are not present in the job execution environment
+     * E.g defaultIdValue will be added to the tags maps for the cluster_id key in case cluster_id
+     * does not exist in jobExecEnv
+     *
+     * @param tags The map of tags
+     * @param jobExecEnv Genie job execution env
+     * @param defaultIdValue Default value for missing id in jobExecEnv
+     */
+    public static void addCommonMetricTags(final JobExecutionEnvironment jobExecEnv,
+                                           final Map<String, String> tags, final String defaultIdValue) {
+        addCommonMetricTags(jobExecEnv.getCluster(), jobExecEnv.getCommand(), tags, defaultIdValue);
+    }
+
+    /**
+     * Convenience method to add common tags for slicing and dicing atlas metrics from Cluster
+     * and Command to the map of tags passed in.
+     * Populates defaultIdValue in case ids are not present in the objects passed in
+     * E.g defaultIdValue will be added to the tags maps for the cluster_id key in case cluster_id
+     * does not exist in Cluster object
+     *
+     * @param tags The map of tags
+     * @param cluster Cluster chosen for running the command
+     * @param command Command chosen to be executed
+     * @param defaultIdValue Default value for missing id in jobExecEnv
+     */
+    public static void addCommonMetricTags(final Cluster cluster, final Command command,
```

Comment:
Same for this, it's adding a specific set of tags, the name of this method could reflect this.